examples/resources/umaa_521/UMAA/EO/AnchorStatus/AnchorReportType.xml

```diff
@@ -8,7 +8,7 @@
     <module name="AnchorStatus">
       <const name="AnchorReportTopic" type="string" value="&quot;UMAA::EO::AnchorStatus::AnchorReport&quot;"/>
       <struct name= "AnchorReportType">
-        <member name="ropeLengthPaidOut" type="nonBasic"  nonBasicTypeName= "UMAA::Common::Measurement::Distance"/>
+        <member name="rodeLengthPaidOut" type="nonBasic"  nonBasicTypeName= "UMAA::Common::Measurement::Distance"/>
```

Comment:
Add a comment mentioning the typo

Reply:
This will be regenerated from an updated IDL set